frontend/src/metabase/home/homepage/components/HomeGreeting/HomeGreeting.tsx

```diff
@@ -35,7 +35,7 @@ const HomeGreeting = ({
   );
 };
 
-const getMessage = (name: string): string => {
+const getMessage = (name: string | null): string => {
```

Comment:
Fix type after we allow first name and last name to be nullable in `api/user.ts`